gatsbyjs/src/components/Page.js

```diff
@@ -1,11 +1,10 @@
 import React from "react";
```

Comment:
I think we can remove this import since `React` is not being used in this file.

Reply:
Good point! I'll fix it now

Reply:
Added back because of the error from the comment above.